core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -192,4 +229,16 @@ private void fillConfMap(Map<PropertyKey, Map<String, List<String>>> targetMap,
       }
     }
   }
+
+  /**
+   * Checks if the time gap between current time and the last time that
+   * we logged the configuration report is bigger than the maximum log interval.
```

Comment:
if we're checking that the time gap is bigger, that means it's a minimum log interval